src/App.css

```
@@ -1,5 +1,6 @@
 .app {
-  height: 100vh;
+  /* height: 100vh; */
```

agregar un comentario TODO para recordar descomentar esto cuando ya no haga falta.
```suggestion
  /*TODO: descomentar al terminar LocalProfile*/
  /* height: 100vh; */
```